tests/integration/test.ts

```diff
@@ -65,5 +65,21 @@ for (const dirent of fs.readdirSync(TEST_FIXTURES_ROOT, {
         }
       }
     })
+
+    it("should NOT fail when running tsc", () => {
+      try {
+        cp.execSync(`npx tsc`, { cwd: TEST_CWD })
+
+        assert.fail("Expect error")
```

Comment:
Hmm. I thought the command should succeed. Was it wrong?

Reply:
It is passing but for some reason it's landing on the catch block. I'll take another look.

Reply:
Oh, I get it! I was writing the test case incorrectly.
My previously written test case should have been:

```diff
      try {
        // ...
-        fs.writeFileSync(actualFile, `no error:${res}`)
      } catch (e: any) {
        // ...
+        return
      }
+      assert.fail("Expect error")
```

The only test case we'll probably add is:

```js
    it("should NOT fail when running tsc", () => {
      cp.execSync(`npx tsc`, { cwd: TEST_CWD })
    })
```

Reply:
Yeah just realize that as well, pushed a changed.